### PR TITLE
Adds pcre to the hunterized and hunter addable libraries.

### PR DESCRIPTION
**Pull request by daminetreg**
Hi @ruslo,

Hope you are well. :smile:

I've added [PCRE](http://www.pcre.org/) to hunter, you might be interested ? Here is the PR for hunter, and [there](https://github.com/daminetreg/pcre/compare/master...hunter) the repository to transfer under hunter-packages.

The dependencies that PCRE needs were already in hunter :smile:, but this makes PCRE loadable from hunter.

I'll add a wiki page from the example in case this PR get merged.

In the example I didn't need to compile with POSITION_INDEPENDENT_CODE on, but in a project where I use pcre I had to recompile pcre with. I believe this is each projects local issue, is it ok to assume that ?

Cheers,

### CODE REVIEW

**Comment by ruslo** [comment]
> and there the repository to transfer under hunter-packages

Created: https://github.com/hunter-packages/pcre

File: examples/pcre/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,15 @@
+cmake_minimum_required (VERSION 3.0)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
+
+project(dowload-pcre)
+
+hunter_add_package(pcre)
+find_package(pcre)
```
Comment (ruslo) [inline examples/pcre/CMakeLists.txt]: You can add `REQUIRED` here. Just to be sure.

**Comment by ruslo** [comment]
See notes about config: https://github.com/hunter-packages/pcre/commit/161a8af09828e9d5909b5e740cbb4c680347678d

**Comment by ruslo** [comment]
> In the example I didn't need to compile with POSITION_INDEPENDENT_CODE on, but in a project where I use pcre I had to recompile pcre with

Are you building on Linux? What type of library, static/shared?

Related discussions:
* https://github.com/ruslo/hunter/issues/63
* https://github.com/ruslo/hunter/issues/64

**Comment by daminetreg** [comment]
Thank you I'll fix these issues and notify you when done. :smile:

**Comment by Cyberunner23** [comment]
@daminetreg Any updates?

**Comment by ruslo** [comment]
Seems abandoned. Feel free to reopen on updates.

**Pull request closed without merging**